README.md

```diff
@@ -26,15 +28,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+They perform the same function, but only 'data-ng-click' will pass HTML validation.
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+I would go with option A because it's the only one that will include the entire body as the root of the application.
 ```
 
```

Comment:
C is preferable, since there could be multiple apps on a page, and often are, but A is a sensible answer. In practice, many angular apps are often on a page.